python/torch_mlir/_torch_mlir_custom_op_example/torch_mlir_custom_op_example.cpp

```diff
@@ -0,0 +1,14 @@
+// For writing an extension like this one, see:
+// https://pytorch.org/tutorials/advanced/torch_script_custom_ops.html
+
+#include <torch/script.h> // One-stop header for PyTorch
+
+torch::Tensor nop(torch::Tensor t) {
```

Comment:
strictly speaking this is an `identity`, not a `nop` -- might make it a bit clearer.